drivers/include/sx126x.h

```diff
@@ -92,6 +92,23 @@ uint32_t sx126x_get_channel(const sx126x_t *dev);
  */
 void sx126x_set_channel(sx126x_t *dev, uint32_t freq);
 
+/**
+ * @brief   Converts symbol value to milliseconds.
+ *
+ * @param[in] dev                      Device descriptor of the driver
+ * @param[in] symbols                  Symbols
+ *
+ * @return Time for symbols in milliseconds
+ */
+static inline int sx126x_symbol_to_msec(sx126x_t *dev, uint16_t symbols)
+{
+    assert((dev->mod_params.bw <= 0x06) && dev->mod_params.bw >= 0x04);
+
+    /* Refer section 4.1.1.7  Time on air in SX1276/77/78/79 datasheet */
+    return (symbols * ((1 << dev->mod_params.sf) \
+                               / (125 * (1 << (dev->mod_params.bw - 4)))));
```

Comment:
You can simplify the equation even more:
```suggestion
    return (symbols * (1 << (dev->mod_params.sf + 7 - dev->mod_params.bw)) / 1000;
```
Since (1/125) = 2^(-3)/1000